### PR TITLE
Karla rubiano programmatic UI

### PR DESCRIPTION
**Pull request by kdrubianoc**
#### What does this PR do?
Changed Storyboards to a programmatic UI.

#### Where should the reviewer start?
Scene Delegate, TableViewController, SearchTableViewController, ViewController, MovieInfoViewController, ReviewsViewController, ReviewsDetailViewController, PersonDetailViewController, Info.plist

#### Code review checklist [Code Review Rubric](https://github.com/wizeline/iOS-apprenticeship-program-2020/blob/main/code-review-rubric.md)

1. DRY Principle (Don’t repeat yourself)

    - [ ] Excellent    
    - [ ] Good    
    - [ ] Adequate
    - [ ] Developing

2. Yagni Principle (You Aren't Gonna Need It)

    - [ ] Excellent    
    - [ ] Good    
    - [ ] Adequate
    - [ ] Developing

3. KISS Principle (Keep It Simple, St...d)

    - [ ] Excellent    
    - [ ] Good    
    - [ ] Adequate
    - [ ] Developing

4. Remove unused code

    - [ ] Excellent    
    - [ ] Good    
    - [ ] Adequate
    - [ ] Developing

5. Hard-coded values

    - [ ] Excellent    
    - [ ] Good    
    - [ ] Adequate
    - [ ] Developing

6. Language style guide & coding conventions
 
 - [Wizeline Style Guide](https://github.com/wizeline/mobile-documentation/blob/master/guidelines/swift-style-guide.md) 
 - [RayWenderlich Style Guide](https://github.com/raywenderlich/swift-style-guide)

    - [ ] Excellent    
    - [ ] Good    
    - [ ] Adequate
    - [ ] Developing

7. Architecture / Design Pattern

    - [ ] MVC  
    - [ ] MVP 
    - [ ] MVVM 
    - [ ] VIPER 

8. Architecture Implementation

    - [ ] Excellent    
    - [ ] Good    
    - [ ] Adequate
    - [ ] Developing

9. SOLID

- Single Responsibility Principle 

    - [ ] Excellent    
    - [ ] Good    
    - [ ] Adequate
    - [ ] Developing

- Open Closed Principle

    - [ ] Excellent    
    - [ ] Good    
    - [ ] Adequate
    - [ ] Developing

- Liskov substitutability principle

    - [ ] Excellent    
    - [ ] Good    
    - [ ] Adequate
    - [ ] Developing

- Interface segregation

    - [ ] Excellent    
    - [ ] Good    
    - [ ] Adequate
    - [ ] Developing

- Dependency Inversion principle

    - [ ] Excellent    
    - [ ] Good    
    - [ ] Adequate
    - [ ] Developing

10. Testing

    - [ ] Excellent    
    - [ ] Good    
    - [ ] Adequate
    - [ ] Developing

11. Force unwrapping !

    - [ ] Excellent    
    - [ ] Good    
    - [ ] Adequate
    - [ ] Developing

12. Mutable vs immutable property (Right use of `let`  and `var`)

    - [ ] Excellent    
    - [ ] Good    
    - [ ] Adequate
    - [ ] Developing

13. Retain cycles (Capture list)

    - [ ] Excellent    
    - [ ] Good    
    - [ ] Adequate
    - [ ] Developing

14. Code Documentation

    - [ ] Excellent    
    - [ ] Good    
    - [ ] Adequate
    - [ ] Developing

### CODE REVIEW

File: TheMovieDb/Controller/ReviewsDetailViewController.swift [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        view.addSubview(authorLabel)
+        view.backgroundColor = .white
+    }
+    
+    private func setupReviewCompletedText() {
+        reviewCompletedText.translatesAutoresizingMaskIntoConstraints = false
+        reviewCompletedText.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 10).isActive = true
+        reviewCompletedText.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor, constant: 20).isActive = true
+        view.safeAreaLayoutGuide.trailingAnchor.constraint(equalTo: reviewCompletedText.trailingAnchor, constant: 20).isActive = true
+//        reviewCompletedText.heightAnchor.constraint(equalToConstant: 100).isActive = true
```
Comment (Carlos-Barron) [inline TheMovieDb/Controller/ReviewsDetailViewController.swift:37]: Don't leave commented code

File: TheMovieDb/Controller/ReviewsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+//  Created by Karla Rubiano on 1/11/21.
+//
+
+import UIKit
+
+final class ReviewsViewController: UIViewController  {
+    
+    var movieID: Int?
+    var movie: Movie?
+    var reviews: [ReviewsDetails] = []
```
Comment (Carlos-Barron) [inline TheMovieDb/Controller/ReviewsViewController.swift:14]: Internal properties?

**Review by Carlos-Barron — APPROVED** [review]

File: TheMovieDb/Controller/ReviewsDetailViewController.swift (hunk 1 above)
Comment (kdrubianoc) [reply TheMovieDb/Controller/ReviewsDetailViewController.swift:37]: Removed. Thanks for pointing that out.

File: TheMovieDb/Controller/ReviewsViewController.swift (hunk 2 above)
Comment (kdrubianoc) [reply TheMovieDb/Controller/ReviewsViewController.swift:14]: Changed `var movie` and `var reviews` to private.

**Pull request merged**